src/utils/fuzzySearch.js

```diff
@@ -19,6 +19,9 @@
  * @returns {boolean} If a match is found
  */
 const fuzzySearch = (search: string, content: string, minCharacters: number = 3, maxGaps: number = 2) => {
+    if (typeof search !== 'string' || typeof content !== 'string') {
```

Comment:
Wouldn't `search` and `content` guaranteed to be strings due to typing?

Reply:
In theory yeah, but I ran into the case when testing so I added a safety clause to be safe.

Reply:
Most of the test files do not include `// @flow` so they don't reveal any type checking issues. this'll let the test cases violate the type contract. 

Do we want to support `search` or `content` being anything but strings?